R/getTimeDivergence.R

```diff
@@ -43,14 +43,16 @@
 #' library(SummarizedExperiment)
 #'
 #' data(hitchip1006)
-#' se <- hitchip1006
+#' se <- transformSamples(hitchip1006, "relabundance")
```

Comment:
There is only one abundance (counts) in hitchip1006 data, so it throws error in this way. Should I change back to counts?

Reply:
The idea is to add relabundances to hitchip data. Can you modify it so?

Reply:
Done